main/ertp/guide/create-specific-erights.md

```diff
@@ -0,0 +1,87 @@
+# Creating a specific eright with ERTP
+
+## Definition of "specific"
+
+In the context of ERTP and assets, "specific" means "non-fungible".
+
+Assets of the same assay are not fungible if each different item of the assay
+cannot be replaced with another.
+House ownership rights are not fungible, because each ownership right refers to a 
+specific house and one house is not equal to another.
+
+## Creating a fungible asset with ERTP
+
+### Creating and sending the asset
+
+In ERTP, digital assets are created by a [`mint`](./mint.html). Having access to the mint
+gives you the power to create more digital assets of the same type at
+will.
+
+Let's say we own an Opera and want to sell tickets to seats for ballet shows. Tickets are
+the specific assets we want to represent. Tickets refer to a specific seat for a specific 
+show at a specific time and date.
+
+To do that, you would first install the [ertp JavaScript package](https://www.npmjs.com/package/@agoric/ertp)
+(`npm install @agoric/ertp`) and then:
+
+```js
+import { makeMint } from '@agoric/ertp/core/mint';
+
+import harden from '@agoric/harden';
+
+import { noCustomization } from '@agoric/ertp/core/config/noCustomization.js';
+import { makeCoreMintKeeper } from '@agoric/ertp/core/config/coreMintKeeper';
+
+import { insist } from '@agoric/ertp/util/insist';
+import { mustBeComparable } from '@agoric/ertp/util/sameStructure';
+
+const insistOptDescription = optDescription => {
+  insist(!!optDescription)`optDescription must be truthy ${optDescription}`;
+  mustBeComparable(optDescription);
+};
+
+function makeInviteConfig() {
+  return harden({
+    ...noCustomization,
+    makeMintKeeper: makeCoreMintKeeper,
+    extentOpsName: 'uniExtentOps',
+    extentOpsArgs: [insistOptDescription],
+  });
+}
```

Comment:
Your suggestion to have the defaults pre-made in `mint.js` is a really good one, given how much has to be imported here that isn't important to the tutorial. Thanks for suggesting that. I'll add it as an issue.